Add reconfig flow to Airobot integration (#42018)
home-assistant/home-assistant.github.io: source/_integrations/airobot.markdown

Host:
    description: "The hostname or IP address of your Airobot thermostat. You can find it in your router settings, or use the hostname format `airobot-thermostat-t01xxxxxx` (replace `t01xxxxxx` with your Device ID in lowercase)."
Device ID:
    description: "The thermostat Device ID (e.g., T01XXXXXX). You can find this in the thermostat menu under **Connectivity** > **Mobile app** screen. This is the same credential used to pair the mobile app."
Password:
    description: "The thermostat password. You can find this in the thermostat menu under **Connectivity** > **Mobile app** screen. This is the same credential used to pair the mobile app."
{% endconfiguration_basic %}

## Reconfiguration

If you need to update the connection settings for your thermostat (such as changing the IP address, Device ID, or password), you can reconfigure the integration without removing and re-adding it:

1. Go to {% my integrations title="**Settings** > **Devices & services**" %}.
2. On the **Airobot** integration, select the three-dot menu and choose **Reconfigure**.
3. Update the connection settings as needed.
4. Select **Submit** to save the new settings.

This is useful when:

> - Your thermostat's IP address has changed (e.g., after router restart or DHCP lease renewal).

Reviewer: ```suggestion
- Your thermostat's IP address has changed (for example, after a router restart or a DHCP lease renewal).
```
